### ISSUE TITLE
[Jenkins Project] Request to host email server for the domain `jenkins.io` by the Linux Foundation

### ISSUE THREAD

**Issue by dduportal** [cdfoundation/foundation#374]
Hello there 👋 

As per the (public) discussion https://github.com/jenkins-infra/helpdesk/issues/2786, the Jenkins project would want to have their email service, for the domain `jenkins.io` to be hosted by the Linux foundation.

It would ensure continuity as the infrastructure officer of the projects may change yearly (elections terms): it's myself for now, but can be someone in the future. For instance it seems that Mailgun is hosting, but the former infrastructure officers does not have access to it (probably set up by Kohsuke a long time ago).

I've opened an issue diredctly to the Linux Foundation (https://jira.linuxfoundation.org/plugins/servlet/theme/portal/2/IT-23864) for reference, but they asked me to check with the "PMO" (Program Manager Officer?) from the CDF: hence this issue.

Thanks in advance!

**Comment by fdegir** [comment]
Thanks for the ping @dduportal!

I am wondering if you could me add to that LFIT ticket so I can see the response and contribute to the discussion on it directly? Just pinged you on slack to pass my mail address.

**Comment by dduportal** [comment]
> 

Yes, no problem @fdegir . I just need to connect to the CDF Slack (I assume I already had an account since you were able to contact me on it). Expect some delays :)

**Comment by dduportal** [comment]
@fdegir done!
